Java/Ch 17. Hard/Q17_21_Volume_of_Histogram/QuestionD.java

```diff
@@ -0,0 +1,67 @@
+package Q17_21_Volume_of_Histogram;
+
+import java.util.Stack;
+
+public class QuestionD {
+
+	public static int computeHistogramVolume(int[] histogram) {
```

Comment:
The complexity in worst case would be 2n, If we represent in Big O it would be O(n).